### ISSUE TITLE
My name got disappeared from the list!

### ISSUE THREAD

**Issue by MatinT-SA** [gayanvoice/top-github-users#222]
Hello,

As of today, I have 265 followers and over 1,000 contributions, which ranked me among the top 50 Iranian GitHub users. However, when I checked the ranking today, my name had unexpectedly disappeared from the list without any apparent reason. Since I was previously included, I'm uncertain what might have caused this issue.

Could you please look into this matter? Thank you in advance.

**Comment by HariSekhon** [comment]
I've had this happen before when I changed my location, the next update run of this project missed me in the locations filter.

Check you are in one of the major city locations the program is hardcoded to recognize.

**Comment by MatinT-SA** [comment]
I also believe in the same thing cause I've recently updated my location and even by assuming that, there's still a bug because I can find at least 10 people on our country's ranking with the same city name that I've recently included, yet those cities aren't in their list.
Therefore, if the system excluded users based on certain city restrictions, those individuals wouldn't appear on the list either, but they do.

It seems like they accept users who meet their requirements, but updating your location becomes an issue if your city is not one of the targeted ones. Given this, they need to fix the issue—either by adding my city to the list or allowing users to update their location without restrictions.

Thanks again for your response.

**Comment by MatinT-SA** [comment]
@HariSekhon 

I also believe in the same thing cause I've recently updated my location and even by assuming that, there's still a bug because I can find at least 10 people on our country's ranking with the same city name that I've recently included, yet those cities aren't in their list.
Therefore, if the system excluded users based on certain city restrictions, those individuals wouldn't appear on the list either, but they do.

It seems like they accept users who meet their requirements, but updating your location becomes an issue if your city is not one of the targeted ones. Given this, they need to fix the issue—either by adding my city to the list or allowing users to update their location without restrictions.

Thanks again for your response.
